packages/zoe/src/contracts/callSpread/fundedCallSpread.js

```diff
@@ -36,7 +36,8 @@ import { Position } from './position.js';
  *
  * terms include:
  * `timer` is a timer, and must be recognized by `priceAuthority`.
- * `expiration` is a time recognized by the `timer`.
+ *  // TODO: is 'timer' a TimerBrand ?
```

Comment:
timer is currently a TimerService, but should migrate (a la #5799) to a TimerBrand.

Reply:
Yep, but not in this PR, given our new limited goals. Deferring to #5799